test/test_image_info.py

```diff
@@ -109,7 +109,8 @@ def test_v2_esp32c3(self):
         hdr = read_image("bootloader_esp32c3.bin")[:8]
         ex_hdr = read_image("bootloader_esp32c3.bin")[8:24]
         assert f"Segments: {hdr[1]}" in out, "Wrong num of segments"
-        assert f"WP pin: {ex_hdr[0]:#02x}" in out, "Wrong WP pin"
+        if ex_hdr[0] != 0xee:
```

Comment:
Can you please change the hex value to uppercase - `0xEE`? This caused the checks to fail.